HW_3/forms.py

```diff
@@ -0,0 +1,11 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, SubmitField
+from wtforms.validators import DataRequired, Email, Length, EqualTo
+
+
+class RegistrationForm(FlaskForm):
+    username = StringField('Username', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(), Email()])
```

Comment:
Можно использовать `EmailField`
```suggestion
    email = EmailField('Email', validators=[DataRequired(), Email()])
```